src/components/ReportWelcomeText.tsx

```diff
@@ -1,53 +1,52 @@
 import React, {useMemo} from 'react';
```

Comment:
<img width="679" alt="Screenshot 2024-10-01 at 16 00 44" src="https://github.com/user-attachments/assets/9d24f97d-ae9f-436a-ab64-fcd5fe4e5a1a">

Copy:
> This is where [submitter] will submit expenses to [workspaceName]. Just use the + button.

Workspace chat renders additional text at the bottom: `Use the + button to split, submit, or track an expense`.